CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format of this document is based on [Keep a Changelog](https://keepachangelo
 
 ## [4.7.0] - 2023-06-05
 
+### Added
+
+- The `gazebo_yarp_forcetorque` plugin now exposes also the `yarp::dev::ITemperatureSensors` interface, so it can stream fake temperature values (https://github.com/robotology/gazebo-yarp-plugins/pull/656).
```

Comment:
Can you move this in the `[Unreleased]`  section? Thanks!

Reply:
Oops, my bad!

Fixed in https://github.com/robotology/gazebo-yarp-plugins/pull/656/commits/a56ab095cf2122b9086125e6f14fdd2874b8f000